### PR TITLE
fix: delete active wallet

### PR DESCRIPTION
**Pull request by BaojunCZ**

### CODE REVIEW

File: packages/neuron-wallet/src/controllers/wallets.ts [hunk 1]
```diff
@@ -177,6 +177,7 @@ class WalletsController {
       const success = WalletsController.service.delete(id)
       if (success) {
         windowManage.broadcast(Channel.Wallets, WalletsMethod.GetAll, WalletsController.getAll())
+        windowManage.broadcast(Channel.Wallets, WalletsMethod.GetActive, WalletsController.getActive())
```
Comment (ashchan) [inline packages/neuron-wallet/src/controllers/wallets.ts]: Are we 100% sure these two events are broadcast and processed in the exact first `getAll` then `getActive` order?

File: packages/neuron-wallet/tests/walletservice.test.electronts [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import assert from 'assert'
```
Comment (ashchan) [inline packages/neuron-wallet/tests/walletservice.test.electronts]: What does the suffix `electronts` mean?

Comment (BaojunCZ) [reply packages/neuron-wallet/tests/walletservice.test.electronts]: I forget to delete it.It is for wallet service test in my local.

File: packages/neuron-wallet/src/controllers/wallets.ts (hunk 1 above)
Comment (BaojunCZ) [reply packages/neuron-wallet/src/controllers/wallets.ts]: I will try to make them to be one

**Pull request merged**